README.md

```diff
@@ -1,9 +1,124 @@
-This is a [demo app](https://web.dev/porting-libusb-to-webusb/) running gPhoto2 on the Web:
-
-![A picture of DSLR camera connected via a USB cable to a laptop. The laptop is running the Web demo mentioned in the article, which mirrors a live video feed from the camera as well as allows to tweak its settings via form controls.](https://web-dev.imgix.net/image/9oK23mr86lhFOwKaoYZ4EySNFp02/MR4YGRvl0Z9AWT6vv3sQ.jpg?auto=format&w=1600)
+# Web-gPhoto2
+A gPhoto2 implementation using WebAssembly to control DSLR cameras from the browser.
 
 Powered by a [custom fork](https://github.com/RReverser/libgphoto2) of [libgphoto2](https://github.com/gphoto/libgphoto2), the [WebUSB](https://github.com/WICG/webusb) backend of [libusb](https://github.com/libusb/libusb), and WebAssembly via [Emscripten](https://emscripten.org/).
 
+# NPM
+## Installation
+```bash
+npm install web-gphoto2
+// or
+yarn add web-gphoto2
+```
+
+## Usage
+**Note: This package ONLY runs in the browser context**. It will not work in NodeJS. If you are using a build-tool make sure to dynamically import this package so it is only imported in the browser context.
+
+```ts
+let initModule: any;
+
+if (typeof window !== 'undefined') {
+    import('web-gphoto2').then((module) => {
+        initModule = module.default;
+    }).catch((error) => {
+        console.error("Error loading web-gphoto2", error);
+    });
+} else {
+    console.warn("web-gphoto2 is only available in the browser");
+}
+```
+
+A short example on how to use this package to obtain the camera config:
```

Comment:
This probably can be updated now that you wrapped / automated away those scheduling calls?

Reply:
Hmm it seems that the version you were reviewing did not have all the changes because I *did* remove that from the readme on my side. Anyways it should be fixed now, including an updated example.